store/sqlstore/post_store.go

```diff
@@ -371,7 +371,7 @@ func (s *SqlPostStore) InvalidateLastPostTimeCache(channelId string) {
 func (s *SqlPostStore) GetEtag(channelId string, allowFromCache bool) store.StoreChannel {
 	return store.Do(func(result *store.StoreResult) {
 		if allowFromCache {
-			if cacheItem, ok := s.lastPostTimeCache.Get(channelId); ok && cacheItem.(int64) > 0 {
+			if cacheItem, ok := s.lastPostTimeCache.Get(channelId); ok {
```

Comment:
What's the reason for this change? I don't understand why that extra check was there in the first place, but I'm not sure why it would we removed